gensim/test/test_fasttext_wrapper.py

```diff
@@ -120,6 +122,37 @@ def testLoadFastTextFormat(self):
         self.assertEqual(self.test_model.wv.syn0_all.shape, (self.test_model.num_ngram_vectors, model_size))
         self.model_sanity(model)
 
+    def testLoadFastTextNewFormat(self):
+        #Test model successfully loaded from fastText (new format) .vec and .bin files
+        new_model = fasttext.FastText.load_fasttext_format(self.test_new_model_file)
+        vocab_size, model_size = 1763, 10
+        self.assertEqual(self.test_new_model.wv.syn0.shape, (vocab_size, model_size))
```

Comment:
Yes, so why not for the other assert statements? If I understand correctly, the whole point of the test is to make sure the model we loaded (`new_model`) has been loaded correctly.